### PR TITLE
lightning: update lightning example binary version and download link #5897

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Updated the version information from v4.0 to v5.0

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5897
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by glorv** [comment]
LGTM

File: get-started-with-tidb-lightning.md [hunk 1]
```diff
@@ -44,17 +44,17 @@ After executing this command, the full backup data is exported to the `/data/my_
 
 ### Step 1: Deploy TiDB cluster
 
-Before the data import, you need to deploy a TiDB cluster (later than v2.0.9). In this tutorial, TiDB v4.0.3 is used. For the deployment method, refer to [TiDB Introduction](/overview.md).
+Before the data import, you need to deploy a TiDB cluster. In this tutorial, TiDB v5.0.0 is used as an example. For the deployment method, refer to [TiDB Introduction](/overview.md).
```
Comment (TomShawn) [inline get-started-with-tidb-lightning.md:47]:
```suggestion
Before the data import, you need to deploy a TiDB cluster. In this tutorial, TiDB v5.0.0 is used as an example. For the deployment method, refer to [Deploy a TiDB Cluster Using TiUP](/production-deployment-using-tiup.md).
```

File: get-started-with-tidb-lightning.md [hunk 2]
```diff
@@ -44,17 +44,17 @@ After executing this command, the full backup data is exported to the `/data/my_
...
 
 Download the TiDB Lightning installation package from the following link:
 
-- **v4.0.3**: [tidb-toolkit-v4.0.3-linux-amd64.tar.gz](https://download.pingcap.org/tidb-toolkit-v4.0.3-linux-amd64.tar.gz)
+- **v5.0.0**: [tidb-toolkit-v5.0.0-linux-amd64.tar.gz](https://download.pingcap.org/tidb-toolkit-v5.0.0-linux-amd64.tar.gz)
 
 > **Note:**
 >
-> Choose the same version of TiDB Lightning as that of the TiDB cluster.
+> TiDB Lightning is compatible with TiDB clusters in lower versions. It is recommended that you download the latest stable version of the TiDB Lightning installation package.
```
Comment (TomShawn) [inline get-started-with-tidb-lightning.md:57]:
```suggestion
> TiDB Lightning is compatible with TiDB clusters of earlier versions. It is recommended that you download the latest stable version of the TiDB Lightning installation package.
```

File: tidb-lightning/deploy-tidb-lightning.md [hunk 3]
```diff
@@ -78,13 +78,17 @@ This section describes how to [deploy TiDB Lightning manually](#deploy-tidb-ligh
...
 You can find deployment instructions in [TiDB Quick Start Guide](/quick-start-with-tidb.md).
 
 #### Step 2: Download the TiDB Lightning installation package
 
-Refer to the [TiDB enterprise tools download page](/download-ecosystem-tools.md#tidb-lightning) to download the TiDB Lightning package (choose the same version as that of the TiDB cluster).
+Refer to the [TiDB enterprise tools download page](/download-ecosystem-tools.md#tidb-lightning) to download the TiDB Lightning package. 
+
+> **Note:**
+>
+> TiDB Lightning is compatible with TiDB clusters in lower versions. It is recommended that you download the latest stable version of the TiDB Lightning installation package.
```
Comment (TomShawn) [inline tidb-lightning/deploy-tidb-lightning.md:91]:
```suggestion
> TiDB Lightning is compatible with TiDB clusters of earlier versions. It is recommended that you download the latest stable version of the TiDB Lightning installation package.
```

File: tidb-lightning/deploy-tidb-lightning.md [hunk 4]
```diff
@@ -78,13 +78,17 @@ This section describes how to [deploy TiDB Lightning manually](#deploy-tidb-ligh
 
 #### Step 1: Deploy a TiDB cluster
 
-Before importing data, you need to have a deployed TiDB cluster, with the cluster version 2.0.9 or above. It is highly recommended to use the latest version.
+Before importing data, you need to have a deployed TiDB cluster. It is highly recommended to use the latest version.
```
Comment (TomShawn) [inline tidb-lightning/deploy-tidb-lightning.md:81]: "stable" is better?
```suggestion
Before importing data, you need to have a deployed TiDB cluster. It is highly recommended to use the latest stable version.
```

**Pull request merged**